src/sources/splunk_hec/mod.rs

```diff
@@ -1316,6 +1320,25 @@ mod tests {
         assert!(event.metadata().splunk_hec_token().is_none());
     }
 
+    #[tokio::test]
+    async fn root() {
+        let message = "root";
+        let (source, address) = source(None).await;
+
+        assert_eq!(200, post(address, "services/collector", message).await);
```

Comment:
Ah, just noticed this test is failing. This should use `channel_n` or similar since `post` sends the raw string message and `/services/collector/` requires JSON.

Reply:
Aha yes, fixed in [be25864](https://github.com/vectordotdev/vector/pull/11941/commits/be258645f9d4127fbab8162b92208cfb24f213df) to send JSON.